web/courses_test.go

```diff
@@ -171,7 +173,11 @@ func TestEnrollmentProcess(t *testing.T) {
 			}
 		}
 	}
-	if diff := cmp.Diff(pendingUserEnrollment, pendingCourseEnrollment, protocmp.Transform()); diff != "" {
+	if diff := cmp.Diff(pendingUserEnrollment, pendingCourseEnrollment, cmp.Options{
```

Comment:
You don't need the `cmp.Options{}` thing; this works fine too since `Diff` takes a variadic argument...
```go
	if diff := cmp.Diff(pendingUserEnrollment, pendingCourseEnrollment,
		protocmp.Transform(),
		protocmp.IgnoreFields(&qf.Enrollment{}, "course"),
	); diff != "" {
```